packages/@okta/vuepress-site/guides/sign-into-spa/section02/angular/define-route.md

```diff
@@ -0,0 +1,20 @@
+Define route that handles a path like `/implicit/callback`. Here's how to do it in [Angular Router](https://angular.io/guide/router):
+
+```javascript
```

Comment:
@aarongranick-okta I struggled to find a way to express to the reader "This goes in your root module file". I thought about putting a comment like `// app.main.js` at the top - any better ideas?

Reply:
Since we are assuming you have an existing app, we assume you have routes defined (somewhere). Our sample quickstart/walkthrough do show/assume a specific app structure, but in this case the developer should know how/where they are defining routes.